README.md

```diff
@@ -43,10 +43,11 @@ You can tell logspout to ignore specific containers by setting an environment va
 
 Using the [httpstream module](http://github.com/gliderlabs/logspout/blob/master/httpstream), you can connect with curl to see your local aggregated logs in realtime. You can do this without setting up a route URI.
 
-	$ docker run -d -p 8000:8000 \
-		-v=/var/run/docker.sock:/var/run/docker.sock \
-		progrium/logspout
-	$ curl $(docker port `docker ps -lq` 8000)/logs
```

Comment:
So you want to publish on 127.0.0.1 but curl localhost? I think consistency should win over ease of typing, especially since this will be mostly for reading purposes and possibly copypasta.